src/universe.rs

```diff
@@ -154,6 +163,30 @@ impl Universe {
         self.atoms.insert(name.to_string(), m);
     }
 
+    /// Merge universe into inself
+    pub fn merge(&mut self, unis: &mut Vec<Universe>) {
```

Comment:
@Karina5005 I think it would be better to make `merge()` take only one universe, not a list of them.

Reply:
Fixed